aait/web/group-4/starter-project/components/homepage/A2SVPrograms.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+import { useState } from "react";
+import A2SVProgram from "./A2SVProgram";
+
+const A2SVPrograms = () => {
+  // Boolean variable to that controls the image to be rendered to the left or to the right
+  const isRight = useState(false);
+  const programs = [
```

Comment:
Instead if having a list here create a json file to store these and import them.